### DISCUSSION TITLE
School does not provide an edu email

### DISCUSSION THREAD

**Discussion by SiddDevZ** [community/community#133113] [GitHub Education]
### Hi everyone,

I am trying to get the github developer student pack from many days, and i always get stuck on 
```"We require applicants to use a school-issued email address to apply. "```

My school does not provide an school-issued email, and i have enough proof to back that i am a student there, how can i get rid of this error? what can i do? please reply.

**Comment by Irash-Perera** [comment]
Yeah, but if you dont have an edu email, this not gonna work mate. Best thing is to do that is contact github for asking student pack for ur institution. They usually reply within a day. [https://support.github.com/](url)
